repository/CargoPackageManager-IntegrationTests/CGOProjectLoadingTest.class.st

```diff
@@ -104,7 +92,7 @@ CGOProjectLoadingTest >> testProjectLoadingWithIndirectMetacelloDependenciesV1Lo
 		version: '1.0'.
 	projectTwo addDependency: configurationProject.
 	
-	projectOne load.
+	projectOne resolveAndLoad.
```

Comment:
Why resolve and load? Loading should imply resolving... I think the rename should be the other way around. We should call `doLoad` or `basicLoad` to the one that does the effective load...